### PR TITLE
Update enable-mail-flow-for-subdomains.md  // Add Modern Public Folder Mailrouting feature

### PR DESCRIPTION
**Pull request by FriedrichTobiasCC**
> - In order to send mails to Modern Public Folders within your Exchange Online Environment you need to set the domain type to internal relay if the domain contains recipientaddresses assigned to Public Folders. Directory Based Edge Blocking is cannot be used for Modern Public Folders. 

should be added, as this seems to be a common problem

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/manage-accepted-domains/enable-mail-flow-for-subdomains.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ If you have a hybrid environment, with mailboxes hosted both in Exchange Online
 > - If you have a limited number of subdomains, and know all the subdomain names, we recommend setting up each subdomain as an accepted domain in the Microsoft 365 admin center, instead of using the procedures in this topic. By setting up each subdomain separately, you can have finer control over mail flow and can include unique mail flow rules (also known transport rules) for each subdomain. For more information about adding a domain in the Microsoft 365 admin center, see [Add your domain to Microsoft 365](/microsoft-365/admin/setup/add-domain).
 >
 > - In order to enable match subdomains, an accepted domain must be set up as an internal relay domain. For information about setting the domain type to internal relay, see [Manage accepted domains in Exchange Online](manage-accepted-domains.md).
+> 
+> - In order to send mails to Modern Public Folders within your Exchange Online Environment you need to set the domain type to internal relay if the domain contains recipientaddresses assigned to Public Folders. Directory Based Edge Blocking is cannot be used for Modern Public Folders.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/manage-accepted-domains/enable-mail-flow-for-subdomains.md:30]:
```suggestion
> - In order to send email to public folders within your Exchange Online environment, you need to set the domain type to internal relay if the domain contains recipient addresses assigned to public folders. Directory-Based Edge Blocking cannot be used for public folders. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @FriedrichTobiasCC, please commit my suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]
@FriedrichTobiasCC - thank you.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**